networkx/algorithms/cuts.py

```diff
@@ -32,11 +32,11 @@ def cut_size(G, S, T=None, weight=None):
     ----------
     G : NetworkX graph
 
-    S : sequence
-        A sequence of nodes in `G`.
+    S : container
```

Comment:
I'm probably being overly pedantic, but I don't think `container` is the right description for the parameter. In this case, `S` and `T` are passed to `nx.edge_boundary`, which expects Iterables for the corresponding nbunch inputs. In the [Python ABC hierarchy](https://docs.python.org/3/library/collections.abc.html#module-collections.abc), Iterables are derived from containers and this function wouldn't work if you passed in an object that *only* implemented `__contains__`. 

I *think* "collection" is probably the more accurate term, but please double check against the linked abc docs and the other instances to make sure this makes sense.

Reply:
"Collection" makes sense-- I updated the docstring with the new commit. Thanks!